package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "firefox-ios",
```

Comment:
nit: this doesn't really add any value and from what I recall it gets generated with npm 🤔